content/Dev/remote_settings_history.rst

```diff
@@ -0,0 +1,179 @@
+
+But internally in the company — above Tarek (our manager) — we were not able to defend our vision very well. Mozilla didn't seem ready to explore the dangerous grounds of hosting user data, at least not beyond *Firefox Sync*. And meanwhile, *Kinto* was still being mentioned as the Mozilla response to Facebook Parse and Google Firebase (completely irrelevant of course).
+
+Luckily, shortly after, the *Go Faster* initiative was started, with the goal of shipping changes to Firefox faster than the current release cycle. If our system could synchronize user data between devices, then it could obviously do a uni-directional synchronization of read-only data from one admin to millions of clients. That's how we started to use *Kinto* to publish the malicious addons blocklist, the certificates revocation list, and some assets for Firefox Android, independently from release trains.
+
+The Kinto client SDK was now in Firefox, and hundreds of millions of clients were pulling data from our servers at https://firefox.settings.services.mozilla.com
+
+
+Antiquity (2016-2018)
+=====================
+
+We were contributing to Gecko in the ``mozilla-central`` repo, and that was not so common for a team in the Cloud Services org. Mercurial, old-style JS, C++, XPCOM... we were not comfortable at all.
+
+`Ethan <https://github.com/glasserc>`_ joined the team. He implemented the ``chrome.storage`` Web Extension API using *Kinto*, and suddenly we were storing addons preferences of users (carefully encrypted on the client side of course).
+
+*Kinto* now had several use-cases within Firefox. However, clearly, we still hadn't reached any critical mass. It was just another arrow on diagrams.
+
+It was a boring and frustrating phase. Pushing the required configuration changes for each new use case. A lot of back and forth with the Ops for root objects manipulations. Responsibilities were limited.
+
+We knew we could do much more, any piece of configuration or JSON in the repository could be migrated to our system! We could use it for internationalization! But we were not good door-to-door salesmen. Feedback, positive or negative, was almost nonexistent.
+
+The revolutionary ideals of our initial team were fading away. We had to admit that we failed at promoting our vision of a user-facing storage backend (see :ref:Thoughts section). Self hosting, decentralized Web, Tor hidden services, were conversations at night around a drink, not in meeting rooms anymore. I'm pretty sure Mozilla even had a few projects running on Google Firebase at this time.
```

Comment:
Maybe add *legend even suggests a few execs were seriously considering switching to Blink*.

Maybe not :sweat_smile: